client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import tinymce from 'tinymce/tinymce';
+import { renderToString } from 'react-dom/server';
+
+import Gridicon from 'components/gridicon';
+import i18n from 'lib/mixins/i18n';
+
+import menuItems from './menu-items';
+
+const initialize = editor => {
+	menuItems.forEach( item =>
```

Comment:
I suppose these menu items and button is still being "added" (but not used) even when the feature flag is disabled? Worth a check to early return perhaps? ([example](https://github.com/Automattic/wp-calypso/blob/b692cc8b971bb9aebd95a5894d5170a9a8017cd5/client/components/tinymce/plugins/media/advanced/index.jsx#L19-L23))